Cherry pick fuse docs from 7.95.x branch to 7.98.x branch (#2871)
codenvy/che-docs: modules/administration-guide/pages/enabling-fuse-for-all-workspaces.adoc

[id="enabling-fuse-overlayfs-for-all-workspaces"]
= Enabling fuse-overlayfs for all workspaces

> Learn about configuring the workspace's container entrypoint script so that fuse-overlayfs is used for all workspaces using that container.

Reviewer: ```suggestion
Learn how to configure the workspace's container entrypoint script so that you can use fuse-overlayfs for all workspaces using that container.
```
